package.json

```diff
@@ -26,6 +30,7 @@
     "react-native-screens": "^2.4.0",
     "react-native-screenshot-detect": "^1.1.0",
     "react-native-snap-carousel": "^3.8.4",
+    "react-native-svg": "^12.1.0",
     "react-native-swiper": "^1.6.0-nightly.5",
     "react-native-tab-view": "^2.13.0",
     "react-native-vector-icons": "^6.6.0",
```

Comment:
One thing is, we should check what dependencies we are using and probably delete the ones we are not